build/custom-ci-Dockerfile

```diff
@@ -1,8 +1,9 @@
-FROM openshift/origin-release:golang-1.16
+FROM openshift/origin-release:golang-1.17
```

Comment:
note that this image tag `golang-1.17` doesn't actually exists at least in docker hub.

Will check and maybe remove this file if it's not used anywhere.

Reply:
Will delete in a follow up later.

Reply:
looks to be a leftover from ansible days